packages/plugins/LayerChooser/src/language.ts

```diff
@@ -6,7 +6,7 @@ const language: LanguageOption[] = [
     resources: {
       plugins: {
         layerChooser: {
-          backgroundTitle: 'Hintergrundkarte',
+          backgroundTitle: 'Hintergrundkarten',
```

Comment:
It's not really a fix, but a change since there was only one background map before and now I added new once. I would think that this would be part of the entry "Feature: Add new background and specialist data layers added." Would you agree or should I add an extra entry for that?